app/views/transactions/_recent_transactions.html.slim

```diff
@@ -0,0 +1,15 @@
+.col-md-8.offset-md-2
+  table.table.table-striped.table-hover
+    thead
+      tr
+        th[scope="col"] Id
+        th Sum
+        th Date
+        th Comment
```

Comment:
`Id`, `Sum`, `Date`, `Comment` to locales. Don't forget about it 😉

Reply:
👍 I was hurrying to get a review today, sorry.

Reply:
@sinpai no need in a hurry 😄 
we are trying to write clean code, without startup mode 😉